Library/Formula/freetds.rb

```diff
@@ -1,10 +1,8 @@
-require "formula"
-
 class Freetds < Formula
   homepage "http://www.freetds.org/"
-  url "http://mirrors.ibiblio.org/freetds/stable/freetds-0.91.tar.gz"
-  sha1 "3ab06c8e208e82197dc25d09ae353d9f3be7db52"
-  revision 2
+  url "ftp://ftp.freetds.org/pub/freetds/stable/freetds-patched.tar.gz"
```

Comment:
I do have a strong preference for a tarball with a version number in the filename, so that it won't break our hash checking when it's inevitably updated.